packages/graphql/src/schemaTypes/objectTypes/gql-LocalSettings.ts

```diff
@@ -33,6 +33,23 @@ export const LocalSettingsPreferences = objectType({
       },
     })
 
+    t.boolean('isValidBrowser', {
+      resolve: async (_source, _args, ctx) => {
+        try {
+          if (!ctx.coreData.cliBrowser) {
+            return false
+          }
+
+          const browser = await ctx._apis.browserApi.ensureAndGetByNameOrPath(ctx.coreData.cliBrowser)
```

Comment:
I'm not sure if this is acceptable. I am basically using the browserApi to ensure that the cli browser passed is a found browser... It will throw an error otherwise. So I wrap it in a try/catch. If it is caught, the browser is either invalid or something is wrong... so we don't launch if isValidBrowser is false. I believe that `ensureAndGetByNameOrPath` is just [this](https://github.com/cypress-io/cypress/blob/develop/packages/server/lib/browsers/utils.ts#L344) utility function under the hood. I am not sure yet if there are side effects from this but would greatly appreciate some input on if this is ok or not.